src/borg/archiver/prune_cmd.py

```diff
@@ -43,7 +43,7 @@ def inner(a):
 
 
 def quarterly_13weekly_period_func(a):
-    """Group archives by extracting from their creation timestamp their ISO-8601 week numbers"""
+    """Group archives by extracting ISO-8601 13-weekly quarter from their creation timestamp"""
```

Comment:
13-weeks quarter?

Reply:
Rather than 13-weekly? The rule is worded like that so I figured . If the other one is made to say "3-monthly quarter" I think they both read well like this. If you'd rather have it be something else I think "13-weeks quarter" reads a bit clunky (but I don't have any other good suggestions right now).

Reply:
"13-weekly quarter" sounded wrong for me as a non-native speaker. But guess we better rely on a native speaker here.